frontend/resourceadm/components/MergeConflictModal/MergeConflictModal.tsx

```diff
@@ -1,19 +1,12 @@
-import React, { useState } from 'react';
-import { toast } from 'react-toastify';
+import React, { forwardRef } from 'react';
 import classes from './MergeConflictModal.module.css';
 import { useTranslation } from 'react-i18next';
-import { Link, Paragraph, Label } from '@digdir/designsystemet-react';
+import { Link, Paragraph, Label, Modal } from '@digdir/designsystemet-react';
```

Comment:
Is it on purpose that you use the `Modal` and not `StudioModal`? 😄 I can see the name of the PR says it is to replace with `StudioModal`

Reply:
yep, I wrote a comment about this above :D When there is a merge conflict, the user must handle it before editing resources, so we are using a Modal without close button for handling merge conflicts. As far as I can see, StudioModal does not support modals without the upper-right close button yet? I could probably hide the close button with css, but that feels a bit hacky.